src/arch/xtensa/task.c

```diff
@@ -37,7 +37,7 @@
 #include <sof/alloc.h>
 #include <sof/debug.h>
 #include <sof/interrupt.h>
-#include <sof/schedule.h>
+#include <sof/edf_schedule.h>
```

Comment:
Just wondering if we should change the naming for below functions to indicate it's only used by edf_schedule.